wiki/osu!coin/de.md

```diff
@@ -0,0 +1,44 @@
+---
+stub: true
+tags:
+  - april fools
+  - Aprilscherze
+---
+
+# osu!coin
+
+![Mehrere osu!coins](img/coins2x.png)![Ein einzelner osu!coin](img/coin2x.png)
+
+Der **osu!coin** war eine scherzhafte Spielwährung, die sich über Monetarisierungssysteme für andere kostenlose Spiele lustig gemacht hat. Die Währung wurde als Aprilscherz 2015 eingeführt und erlaubte Spieler eine einzige [Beatmap](/wiki/Beatmap) zu spielen, im Austausch gegen einen osu!coin.
+
+## Veröffentlichung
+
+Die [offizielle Ankündigung](https://osu.ppy.sh/home/news/2015-03-31-osucoins) erklärt, dass der osu!coin entwickelt, um ![][flag_AU] [peppy](https://osu.ppy.sh/users/2)s privaten Jet zu finanzieren, da das aktuelle Monetarisierungssystem es ihm nicht erlauben würde, den Jet zu seinen Lebzeiten zu kaufen. Die Ankündigung führt weiter aus, dass jeder neue Spieler mit 10 osu!coins beginnen würde und jeden Tag 10 mehr bekommt. Bereits registrierte Nutzer würden zwei zusätzliche osu!coins als ein Bonus bekommen. Spieler könnten dann mehr osu!coins durch die Erreichung von [Combo](/wiki/Gameplay/Combo_(score_multiplier)) ergattern. Ein einzelner osu!coin würde man für jede 100ter Combo erhalten.
```

Comment:
```suggestion
Die [offizielle Ankündigung](https://osu.ppy.sh/home/news/2015-03-31-osucoins) erklärt, dass der osu!coin entwickelt wurde, um ![][flag_AU] [peppy](https://osu.ppy.sh/users/2)s privaten Jet zu finanzieren, da das aktuelle Monetarisierungssystem es ihm nicht erlauben würde, den Jet zu seinen Lebzeiten zu kaufen. Die Ankündigung führt weiter aus, dass jeder neue Spieler mit 10 osu!coins beginnen würde und jeden Tag 10 mehr bekommt. Bereits registrierte Nutzer würden zwei zusätzliche osu!coins als ein Bonus bekommen. Spieler könnten dann mehr osu!coins durch die Erreichung von [Combo](/wiki/Gameplay/Combo_(score_multiplier)) ergattern. Eine einzelne osu!coin würde man für jede 100ter Combo erhalten.
```

Reply:
I've used the male version for osu!coin (as in `der Coin`) throughout the whole article, so I would like to stick to that.

Might be relevant here: [The Duden says `der oder die Bitcoin` is fine](https://www.duden.de/rechtschreibung/Bitcoin_Zahlungseinheit).

Reply:
Oh damn, how did i miss all the other parts where you used male version xd
Though i would probably use `die Coin` also, because of `die Münze`. 🤔 

Either is correct so I'm fine with whatever.

Reply:
Even though the Duden lists it as valid, saying `die Coin` sounds weird to me. If you're fine with using `der`, could you please edit the suggestion, so I can commit it?

By the way, you could also translate coin with `das Geldstück`, so I'm not sure if this is always applicable, but these are just my two cents (no pun intended lol). I think that it's a personal preference at this point.

Reply:
Yeah pretty much personal preference, I edited the suggestion

Reply:
Thanks 👍